conftest.py

```diff
@@ -27,12 +27,12 @@ def driver(request):
     if ("localhost" in machine) is True:
         hub = config.get('driver', 'hub')
 
-    browser = webdriver.Remote(
-            command_executor="http://" + hub + ":4444/wd/hub",
-            desired_capabilities={
-                "browserName": request.config.getoption("--driver")
-            })
-
+    # browser = webdriver.Remote(
+    #         command_executor="http://" + hub + ":4444/wd/hub",
+    #         desired_capabilities={
+    #             "browserName": request.config.getoption("--driver")
+    #         })
+    browser = webdriver.Chrome(executable_path='/home/rohan/Downloads/chromedriver')
```

Comment:
@rohan2794 please revert this change.

Reply:
change reverted with some other logout fix